time stamp
float64[] rotation
float64[] alpha
float64[] alpha_deg
float64[] BalphaU
float64[] tau
float64[] u